rust/datafusion/src/execution/context.rs

```diff
@@ -564,13 +634,30 @@ impl ExecutionConfig {
         self.optimizers.push(optimizer_rule);
         self
     }
+
+    /// Selects a name for the default catalog and schema
+    pub fn with_default_catalog_and_schema(
```

Comment:
I like this API. 👍 

I wonder if we need an API for getting `default_catalog` and `default_schema`?